### ISSUE TITLE
Find cutsets of strings.{Trim,TrimLeft,TrimRight}, which are not in order

### ISSUE THREAD

**Issue by breml** [go-critic/go-critic#1148]
Cutsets for `strings.{Trim,TrimLeft,TrimRight}` should be in order to prevent bugs where `strings.TrimRight` is confused (or replaced) by `strings.TrimSuffix`.

Before:

```Go
strings.TrimLeft(url, `/prefix`)
```

After:

```Go
strings.TrimLeft(url, `/efiprx`)
```

Inspired by [proposal: cmd/vet warns when the cutset of strings.{Trim,TrimLeft,TrimRight} is not in order](https://github.com/golang/go/issues/47822)

**Comment by cristaloleg** [comment]
Should we just blindly check that `strings.TrimLeft` 2nd param isn't sorted and suggest a replacement or argument fix?

Ideas/objections? @breml @quasilyte

**Comment by breml** [comment]
I guess, for this case we can warn, but we can not always give the correct fix to the user. So I would expect the linter to provide a meaningful error message indicating, that the user either should order the cutset in alphabetical order (maybe provide the already ordered value in the error message) or the user should use `strings.TrimPrefix` or `strings.TrimSuffix` instead.
